mobanfile

```diff
@@ -12,7 +14,7 @@ targets:
   - docs/conf.py: custom_conf.py.jj2
   - .travis.yml: travis.yml
   - requirements.txt: requirements.txt.jj2
-  - .gitignore: moban_gitignore.jj2
+  # .gitignore: moban_gitignore.jj2
```

Comment:
This is https://github.com/moremoban/pypi-mobans/issues/94 .  I am working on that now

Reply:
👍